_includes/payment-order-other-features.md

```diff
@@ -1464,6 +1464,83 @@ with the request parameter `instrument`.
     api_resource="paymentorders"
     documentation_section=include.documentation_section %}
 
+## MOTO
+
+{% include alert-agreement-required.md %}
+
+Mail-Order Telephone-Order
+Notes to ourself: 
+- only supported by Purchase (not Verify), 
+- Is only supported by CreditCard
+- Only valid with 3DS 2.x
+- skips SCA
```

Comment:
Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6175237)